modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/EdgeInformation.js

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayForm, {ClayInput, ClayToggle} from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import PropTypes from 'prop-types';
+import React, {useContext} from 'react';
+import {isEdge} from 'react-flow-renderer';
+
+import {DefinitionBuilderContext} from '../../../../DefinitionBuilderContext';
+import {DiagramBuilderContext} from '../../../DiagramBuilderContext';
+import SidebarPanel from '../SidebarPanel';
+import {isIdDuplicated} from '../utils';
+
+export default function EdgeInformation({errors, setErrors}) {
+	const {defaultLanguageId, selectedLanguageId} = useContext(
+		DefinitionBuilderContext
+	);
+	const {
+		elements,
+		selectedItem,
+		selectedItemNewId,
+		setElements,
+		setSelectedItem,
+		setSelectedItemNewId,
+	} = useContext(DiagramBuilderContext);
+
+	const onToggleDefault = (defaultEdge) => {
+		if (defaultEdge) {
+			setElements((previousElements) =>
+				previousElements.map((element) => {
+					if (
+						isEdge(element) &&
+						element.source === selectedItem.source
+					) {
+						element.data.defaultEdge = false;
+					}
+
+					return element;
+				})
+			);
+		}
+
+		setSelectedItem({
+			...selectedItem,
+			data: {
+				...selectedItem.data,
+				defaultEdge,
+			},
+		});
+	};
+
+	return (
+		<SidebarPanel panelTitle={Liferay.Language.get('transitions')}>
+			<ClayForm.Group className={errors.label ? 'has-error' : ''}>
+				<label htmlFor="nodeLabel">
+					{Liferay.Language.get('label')}
+
+					<span className="ml-1 mr-1 text-warning">*</span>
+
+					<span title={Liferay.Language.get('label-name')}>
+						<ClayIcon
+							className="text-muted"
+							symbol="question-circle-full"
+						/>
+					</span>
+				</label>
+
+				<ClayInput
+					id="edgeLabel"
+					onChange={({target}) => {
```

Comment:
@jotascript Is this too much code to onChange? What do you think about change it for two functions, one to set errors and another to set the selected item?